web/skins/classic/views/js/watch.js

```diff
@@ -335,18 +295,22 @@ function streamCmdPause(action) {
 }
 
 function onPlay() {
-  setButtonState('pauseBtn', 'inactive');
-  setButtonState('playBtn', 'active');
+  //monitorStream.setup_onplay(onPlay); //IgorA100 Added for testing, but probably not required
+  //setButtonState('pauseBtn', 'inactive');
+  //setButtonState('playBtn', 'active');
+  setButtonStateWatch('pauseBtn', 'inactive');
+  setButtonStateWatch('stopBtn', 'inactive');
+  setButtonStateWatch('playBtn', 'unavail');
   if (monitorStream.status.delayed == true) {
-    setButtonState('stopBtn', 'inactive');
+    //setButtonState('stopBtn', 'inactive');
```

Comment:
A lot of this stuff is old.  The fact that the styles were only defined for inputs is because back in the day we used <input type="button">.  At some point, I decided to do away with all that because using <button> Was better.  Clearly I didn't update the css.  I don't think we need three states.  Just enabled/disabled will do.  I do think that if something is disabled, it is useful to make sure the title tooltip says why.  I also hate, but have never gotten around to doing... is remove the stop button.  The play/pause/stop concept was for cassette tapes, where pausing would actually damage/wear out the tape/mechanisms.  In a digital world, the difference is that stop goes back to the beginning of the file and pause doesn't.  The I don't feel that is useful.  Others disagree with me.

Reply:
I have a video cassette recorder and an audio recorder and a vinyl player with records (left over from the 90s) and I sometimes watch and listen to them :) I know what it is. Yes, the digital world has changed many things.
Regarding the Stop/Pause buttons - Pause can only be useful when watching streaming video that is cached. Those. after releasing the Pause button, viewing will continue from the point where it was paused (i.e. not in real time, but with a delay). I don't know if this behavior is provided in ZM or not

> I don't think we need three states. Just enabled/disabled will do

Agree. But now I'm not ready for the "great feat of global redesign" of the DVR control buttons. Maybe later...

Reply:
Here is an example of inactive vs disabled, etc.  The ptzToggle button: should always be visible, unless ptz is disabled globally.  So always visible, but if the monitor doesn't support ptz, then it is disabled.  If the monitor DOES support ptz, then it should be either inactive or active depending on whether we are showing the ptz controls.